### PR TITLE
Integrate dapr runtime v1.8.1 release

### PR DESCRIPTION
**Pull request by emsameen**
# Description

Pin the dap cli and runtime versions for the stable 1.8.0/1.8.1 release.

## Azure DevOps PBI/Task reference

AB#10569
## Checklist - Manual tasks

* [x] Examples are executing successfully
* [ ] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [ ] Created/updated integration tests.
* [x] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

**Comment by DocMoebiuz** [comment]
As discussed we should pin the version to a specific one. If that is 1.8.0 now instead of the RC3 then we should take that one.

Btw, have you found a way to not only specify the runtime version but also the version for the CLI?

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]
pin version to 1.8.0

File: prerequisite_settings.json [hunk 1]
```diff
@@ -16,5 +16,9 @@
     },
     "python": {
         "version": "3.9"
+    },
+    "dapr":{
+        "version": "1.8.0",
+        "runtime": "1.8.1"
```
Comment (mml5bg) [inline prerequisite_settings.json:22]: Please change runtime version in title of PR request, to have consistency

**Review by mml5bg — APPROVED** [review]
I approve this, for consistency, you should change runtime version in title of this PR

File: prerequisite_settings.json (hunk 1 above)
Comment (emsameen) [reply prerequisite_settings.json:22]: done

**Review by DocMoebiuz — APPROVED** [review]
LGTM

**Pull request merged**